utils/eligibility_clause_utils.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"github.com/biter777/countries"
+)
+
+func GetSubdivisionsByCountry(country string) (valid bool, subdivisions []countries.SubdivisionCode) {
```

Comment:
nit: I think it's slightly more idiomatic Go to have this thing's return signature be

`subdivisions []countries.SubdivisionCode, error ` then return `fmt.Errorf("country does not exist")` in the error case, instead of having the bool first.

Reply:
thanks for calling that out. will update